### PR TITLE
🌐 [i18n-KO] Translated `guides/manage-spaces.md` to Korean

### PR DESCRIPTION
**Pull request by boyunJang**
# What does this PR do?

Translated the `guides/manage-spaces.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team Hugging Face KREW, may you please review this PR? 
@harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae, @usr-bin-ksh, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

File: docs/source/ko/guides/manage-spaces.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+>>> api.duplicate_space(
+...     from_id=repo_id,
+...     secrets=[{"key"="HF_TOKEN", "value"="hf_api_***"}, ...],
+...     variables=[{"key"="MODEL_REPO_ID", "value"="user/repo"}, ...],
+... )
+```
+
+**4. 하드웨어 구성**
+
+기본적으로 Space는 무료로 CPU 환경에서 실행됩니다. 하지만 GPU에서 실행하기 위해 하드웨어를 업그레이드할 수 있습니다. 하드웨어를 업그레이드하려면 결제 카드 또는 커뮤니티 그랜트가 필요합니다. 자세한 내용은 [문서](https://huggingface.co/docs/hub/spaces-gpus)를 참조하세요.
```
Comment (seoulsky-field) [inline docs/source/ko/guides/manage-spaces.md:89]:
```suggestion
기본적으로 Space는 무료로 CPU 환경에서 실행됩니다. GPU에서 실행하기 위해 하드웨어를 업그레이드 할 수도 있습니다. 하드웨어를 업그레이드하려면 결제 카드 또는 커뮤니티 그랜트가 필요합니다. 자세한 내용은 [문서](https://huggingface.co/docs/hub/spaces-gpus)를 참조하세요.
```

문법적인 부분  (띄어쓰기나 문장 전환 부분)을 수정해보았습니다!

File: docs/source/ko/guides/manage-spaces.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+>>> api.duplicate_space(
+...     from_id=repo_id,
+...     hardware="t4-medium",
+...     sleep_time="3600",
+... )
+```
+
+**6. Space에 지속적으로 저장소 추가하기**
+
+Space를 다시 시작할 때 지속적으로 디스크 공간에 액세스할 수 있는 원하는 저장소 계층을 선택할 수 있습니다. 이는 기존의 하드 드라이브와 같이 디스크에서 읽고 쓸 수 있음을 의미합니다. 자세한 내용은 [문서](https://huggingface.co/docs/hub/spaces-storage#persistent-storage)를 참조하세요.
```
Comment (seoulsky-field) [inline docs/source/ko/guides/manage-spaces.md:187]:
```suggestion
Space를 다시 시작할 때 지속적으로 디스크 공간에 접근할 수 있는 원하는 저장소 계층을 선택할 수 있습니다. 이는 기존의 하드 드라이브와 같이 디스크에서 읽고 쓸 수 있음을 의미합니다. 자세한 내용은 [문서](https://huggingface.co/docs/hub/spaces-storage#persistent-storage)를 참조하세요.
```

국문 의미가 담기도록 수정하였습니다!

**Review by seoulsky-field — COMMENTED** [review]
좋은 번역 감사하고, 긴 문서 번역하느라 고생하셨습니다!! 문법적인 부분만 조금 수정해보았습니다!

File: docs/source/ko/guides/manage-spaces.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+
+다음은 로컬 폴더 `src/`를 사용자의 컴퓨터에서 Space로 업로드하는 예시입니다:
+
+```py
+>>> api.upload_folder(repo_id=repo_id, repo_type="space", folder_path="src/")
+```
+
+이 단계에서는 앱이 이미 무료로 Hub에서 실행 중이어야 합니다! 그러나 더 많은 보안 정보와 업그레이드된 하드웨어를 이용하여 추가적으로 구성할 수 있습니다.
+
+**3. 보안 정보 변수 구성**
```
Comment (seoyoung-3060) [inline docs/source/ko/guides/manage-spaces.md:42]:
```suggestion
**3. 보안 정보와 변수 설정하기**
```
좀 더 매끄럽게 번역해보았습니다!

File: docs/source/ko/guides/manage-spaces.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+>>> runtime = api.get_space_runtime(repo_id=repo_id)
+>>> runtime.stage
+"RUNNING_BUILDING"
+>>> runtime.hardware
+"cpu-basic"
+>>> runtime.requested_hardware
+"t4-medium"
+```
+
+이제 완전히 구성된 Space가 있습니다. 사용이 완료되면 Space를 다시 "cpu-classic"로 다운그레이드해야 합니다.
```
Comment (seoyoung-3060) [inline docs/source/ko/guides/manage-spaces.md:112]:
```suggestion
이제 완전히 구성된 Space를 가지게 되었습니다. 사용이 끝난 후에는 스페이스를 "cpu-classic"으로 다운그레이드하는 것을 잊지 마세요.
```
좀 더 매끄럽게 읽히도록 수정해보았습니다!

**Review by seoyoung-3060 — COMMENTED** [review]
좋은 번역 감사합니다!
조금 더 매끄럽게 읽히기 위해 세세한 수정만 진행하였습니다.
감사합니다 :)

File: docs/source/ko/guides/manage-spaces.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+>>> runtime = api.get_space_runtime(repo_id=repo_id)
+>>> runtime.stage
+"RUNNING_BUILDING"
+>>> runtime.hardware
+"cpu-basic"
+>>> runtime.requested_hardware
+"t4-medium"
+```
+
+이제 완전히 구성된 Space를 가지게 되었습니다. 사용이 끝난 후에는 스페이스를 "cpu-classic"으로 다운그레이드하는 것을 잊지 마세요.
```
Comment (boyunJang) [inline docs/source/ko/guides/manage-spaces.md:112]:
```suggestion
이제 완전히 구성된 Space를 가지게 되었습니다. 사용이 끝난 후에는 Space를 "cpu-classic"으로 다운그레이드하는 것을 잊지 마세요.
```
스페이스 -> Space로 수정했습니다

**Comment by jungnerd** [comment]
@Wauplin LGTM! This PR is ready to merge!

**Review by Wauplin — APPROVED** [review]
Then let's get it merged! Thanks!

**Pull request merged**